_maps/RandomRuins/SpaceRuins/bus.dmm

```diff
@@ -291,7 +303,8 @@
 /obj/effect/decal/cleanable/dirt,
 /obj/item/toy/plush/awakenedplushie{
 	pixel_z = 26;
-	pixel_y = 1
+	pixel_y = 1;
+	anchored = 1
```

Comment:
should really just fix offsetting not working and let this one come home, since he screams at random (and is I think also not a coder ref unless it's ninja?)